File: Geovis/Testing/Cxx/TestGeoProjection.cxx
#include "vtkGeoProjection.h"

int TestGeoProjection( int, char*[] )
{
  int np = vtkGeoProjection::GetNumberOfProjections();
  cout << "Supported projections:\n";
  for ( int i = 0; i < np; ++ i )
    {
    cout << "Projection: " << vtkGeoProjection::GetProjectionName( i ) << "\n";
    cout << "\t" << vtkGeoProjection::GetProjectionDescription( i ) << "\n";
    }
  cout << "-------\n";
  vtkGeoProjection* proj = vtkGeoProjection::New();
  const char* projName = "rouss";
  proj->SetName( projName );
  cout << projName << " is " << proj->GetDescription() << "\n";
  proj->Delete();
  return 0;
}

File: Geovis/Core/Testing/Python/TestGeoProjection.py
from vtkmodules.vtkGeovisCore import vtkGeoProjection

np = vtkGeoProjection.GetNumberOfProjections()
for i in range(np):
    print("Projection: {}\n\t{}".format(
        vtkGeoProjection.GetProjectionName(i),
        vtkGeoProjection.GetProjectionDescription(i)))
projName = "rouss"
proj = vtkGeoProjection()
proj.SetName(projName)
print("Projection: {}\n\t{}".format(projName, proj.GetDescription()))
